Testscripts/Linux/nginx-community-test.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
which case use this nginx-community-test.sh ?

Reply:
Dropped